incubator/cassandra/templates/petset.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: v1
```

Comment:
Should I separate each kind into individual file e.g. `service.yaml`, `statefulset.yaml` ?